docs: :memo: Move user-test.qmd from Sprout to Teams repo

K-Beicher opened the pull request:
## Description

<!-- 
This template covers all PRs for Seedcase, please note that if you are
submitting a PR for changes to:

a) General documentation you should delete the sections Testing, Code
Documentation, and the first part of the Author Checklist.

b) Code you should delete the second section of the Author Checklist.

Otherwise, delete any sections that don't apply.
-->

<!-- DO NOT LEAVE THIS SECTION BLANK -->

- This PR finds a better home for the user-test.qmd document, moving it from Sprout to Teams.
- These changes are done because we want to use the user test document in more than one dev repo, and the teams repo seems to be the most natural place to place it (at least for now).

## Related Issues

<!-- List issues the PR closes -->

Closes Sprout seedcase-project/seedcase-sprout#424

<!-- Connect this PR to relevant issues, to help the reviewer but also for record-keeping. -->

## Reviewer Focus
<!-- Please delete as appropriate: -->
This PR only needs a quick review.

First time moving a document, please note that the original has not yet been deleted, that will be done once I know the document is sitting correctly in Teams.

<!-- Any particular section the reviewer should focus on, anywhere that would be a good place to start? -->

## Checklist

<!-- This is to help you determine if your work is ready to be reviewed, if an item is not relevant then you can mark it as done (because you have checked and found that it isn't needed) -->

For general documentation:

- [ ] Spell-check
    - [ ] US
    - [ ] UK
- [ ] Did the page(s) preview correctly on your machine without breaking
- [ ] New category words (keywords) (if any) added to the code snippet file

K-Beicher committed:
docs: :memo: Move user-test.qmd from Sprout to Teams repo

signekb reviewed (commented):
Nice 👌 
Should this file be in the `entries` folder instead of creating a new `docs/uer-tests` directory?

K-Beicher commented:
Good point @signekb, @lwjohnst86 where would you put it?

lwjohnst86 commented:
Hmmm, excellent question... I think maybe `user-tests/sprout.qmd`? I'll make some edits anyway, so I can move it over.

lwjohnst86 committed:
chore: :truck: move file to more appropriate location

lwjohnst86 committed:
chore: :truck: mistakenly saved it to wrong location

lwjohnst86 committed:
docs: :art: internal links to headers are not necessary for short docs

Since Quarto builds the website and includes a table of contents on the side, including header links isn't necessary.

lwjohnst86 committed:
docs: :memo: reviewed and made some edits to the user test document

lwjohnst86 committed:
build: :wrench: add the user tests folder to the website navigation

signekb commented on user-tests/sprout.qmd:
```diff
@@ -1,103 +1,108 @@
-|-----|---------------|----------|-----------|
-| 1  | run           | 10       | 01-03-2024 |
-| 2  | walk          | 25       | 02-04-2024 |
+| id  | physical_activity | duration | date       |
+|-----|-------------------|----------|------------|
+| 1   | run               | 10       | 01-03-2024 |
+| 2   | walk              | 25       | 02-04-2024 |
```
@lwjohnst86 Did you do this manually?
